.github/workflows/ci.yml

```diff
@@ -13,3 +13,21 @@ jobs:
 
       - name: lint
         run: helm lint charts/*
+
+  test_primary_site:
+    runs-on: ubuntu-latest
```

Comment:
Should I be using the buildpulse runners here? Are they set up for this repo?

Reply:
@sofuture I don't know, do you?